src/include/duckdb/parser/parsed_data/create_sequence_info.hpp

```diff
@@ -13,6 +13,16 @@
 
 namespace duckdb {
 
+class SequenceInfo {
+public:
```

Comment:
We try to avoid bitmasks unless necessary, could we turn this into an `enum class` instead and move this to the `transform_alter_sequence` file (which is the only place where this is used)?